src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner sc = new Scanner(System.in);
+        printHorizontalLine();
+        System.out.println("  Hello! I'm Duke");
+        System.out.println("  What can I do for you?");
+        printHorizontalLine();
+        line = sc.nextLine();
+        while (!line.equals("bye")){
+            try {
+                if(line.equals("list")){
```

Comment:
To enhance readability, you may want to consider putting a space between `if` and `(`. This also goes for the rest of your `if` and `else if` statements.